src/Core/src/Platform/iOS/MauiContextExtensions.cs

```diff
@@ -1,14 +1,10 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace Microsoft.Maui
+﻿namespace Microsoft.Maui
 {
 	internal static class MauiContextExtensions
 	{
 		public static FlowDirection GetFlowDirection(this IMauiContext mauiContext)
 		{
-			var window = mauiContext.Window;
+			var window = (mauiContext as IMauiWindowContext)?.Window;
```

Comment:
In some cases where windows-y things are needed, we have to cast. But, this is still needed anyway if we were trying to get the flow direction of an app. We could potentially get this, but it may not always make sense.